sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/checks/AndroidBroadcastingCheck.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * SonarSource Kotlin
+ * Copyright (C) 2018-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.kotlin.checks
+
+import org.jetbrains.kotlin.psi.KtCallExpression
+import org.jetbrains.kotlin.resolve.BindingContext
+import org.jetbrains.kotlin.resolve.calls.model.ResolvedCall
+import org.jetbrains.kotlin.resolve.calls.model.ResolvedValueArgument
+import org.sonar.check.Rule
+import org.sonarsource.kotlin.api.CallAbstractCheck
+import org.sonarsource.kotlin.api.FunMatcher
+import org.sonarsource.kotlin.api.isNull
+import org.sonarsource.kotlin.plugin.KotlinFileContext
+
+private const val MESSAGE = "Make sure that broadcasting intents is safe here."
+
+private val MATCHER = FunMatcher(qualifier = "android.content.Context") {
```

Comment:
Really minor, maybe a more descriptive name? E.g. `FUNS_TO_VISIT` or something

Reply:
I think I can even inline it. Since we have a property it won't be recomputed